tensorflow/java/src/test/java/org/tensorflow/OperationBuilderTest.java

```diff
@@ -51,7 +49,7 @@ public void failWhenMixingOperationsOnDifferentGraphs() {
   public void failOnUseAfterBuild() {
     try (Graph g = new Graph();
         @SuppressWarnings("unchecked")
-            Tensor<TFInt32> t = (Tensor<TFInt32>) Tensor.create(1)) {
+            Tensor<Integer> t = (Tensor<Integer>) Tensor.create(1)) {
```

Comment:
I've left the test cases creating tensors in various ways that are currently supported, though user code should probably use the `Tensors` class that doesn't exist yet.